test/db/cmd/print

```diff
@@ -72,6 +72,23 @@ EXPECT=<<EOF
 EOF
 RUN
 
+NAME=MIPS new abi
+FILE=bins/elf/analysis/mips64r2-busybox-loongson
+CMDS=<<EOF
+e asm.arch=mips.gnu
+s 0x120005978
+pd 6
+EOF
+EXPECT=<<EOF
+            0x120005978      1800bfff       sd ra, 24(sp)
```

Comment:
So, is mips64 on capstone broken as well?
```
$ ./cstool/cstool mips64 1800bfff3800a7ff4000a8ff4800a9ff5000aaff5800abff                                                                          
 0  18 00 bf ff  sd     $ra, 0x18($sp)
 4  38 00 a7 ff  sd     $a3, 0x38($sp)
 8  40 00 a8 ff  sd     $t0, 0x40($sp)
 c  48 00 a9 ff  sd     $t1, 0x48($sp)
10  50 00 aa ff  sd     $t2, 0x50($sp)
14  58 00 ab ff  sd     $t3, 0x58($sp)
```
It uses t0-t1 instead of a3-a7. Is it alright?

Reply:
《see mips run》 11.2 Argument Passing and Stack Conventions for MIPS ABIs - > TABLE 11.2 Integer Register Usage Evolution in Newer SGI Tools

|          | o32    | n32/n64         |
| -------- | ------ | --------------- |
| $8–$11   | t0–t3  | a4–a7 Arguments |
| $12–$15  | t4–t7  | t0–t3           |
| $24, $25 | t8, t9 | t8, t9          |

ABI information is needed to determine a4~a7 or t0~t3

Reply:
Perfect, just wanted to be sure ;) Thanks for the info!